context/app/static/js/shared-styles/Links/iconLinks/IconLink/IconLink.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FlexOutboundLink, FlexLightBlueLink, StyledSpan } from './style';
+
+function IconLink({ children, icon, iconPosition, isOutbound, ...rest }) {
+  const LinkComponent = isOutbound ? FlexOutboundLink : FlexLightBlueLink;
+
+  return (
+    <LinkComponent {...rest}>
+      {iconPosition === 'start' && icon}
+      <StyledSpan $iconPosition={iconPosition}>{children}</StyledSpan>
+      {iconPosition === 'end' && icon}
+    </LinkComponent>
+  );
+}
+
+IconLink.propTypes = {
+  icon: PropTypes.element.isRequired,
+  iconPosition: PropTypes.oneOf(['start', 'end']),
```

Comment:
I would vote for making this a boolean, that would only need to be specified when the non-default option is chosen... probably `iconOnLeft` or `iconAtStart`, and the default would be the icon at the end.